JitsiConference.js

```diff
@@ -2086,9 +2087,16 @@ JitsiConference.prototype._onIceConnectionFailed = function(session) {
         // and "bad" connection
         Statistics.analytics.addPermanentProperties({ p2pFailed: true });
 
-        // Log analytics event, but only for the initiator to not count it twice
-        if (this.p2pJingleSession && this.p2pJingleSession.isInitiator) {
-            Statistics.sendEventToAll(P2P_FAILED);
+        if (this.p2pJingleSession) {
+            const event
+                = createP2pEvent(
+                    'failed',
```

Comment:
these bare strings don't feel great to me...i assume we only fire each one of these in a single place (so not a big risk of an accidental typo when trying to send the same event somewhere else?).  even still, using named constants is more descriptive which i think is better. (i.e. P2P_FAILED_EVENT as opposed to 'failed')

Reply:
also if we ever wanted to change the format (e.g. camel case instead of periods) it'd be easier to do since all the constants would (presumably) be defined one place, as opposed to the strings all over the code

Reply:
> i assume we only fire each one of these in a single place

No, not necessarily. 

But I'll add constants